### PR TITLE
All sorts of fixes to get tests back up

### PR DESCRIPTION
**Pull request by daisieh**

### CODE REVIEW

File: chord_metadata_service/restapi/tests/test_argo.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def setUp(self):
         self.primary_diagnosis = valid_cancer_condition()
 
     def test_get_argo(self):
-        create_object_response = get_response("cancercondition-list", self.primary_diagnosis)
+        create_object_response = get_response("cancerconditions-list", self.primary_diagnosis)
```
Comment (amanjeev) [inline chord_metadata_service/restapi/tests/test_argo.py:223]: btw I do not see anything called `cancerconditions-list`.  I see `cancerconditions` though.

File: chord_metadata_service/restapi/tests/test_argo.py [hunk 2]
```diff
@@ -198,7 +198,7 @@ def setUp(self):
         self.specimen = valid_genetic_specimen("specimen:01")
 
     def test_get_argo(self):
-        create_object_response = get_response("geneticspecimen-list", self.specimen)
+        create_object_response = get_response("geneticspecimens-list", self.specimen)
```
Comment (amanjeev) [inline chord_metadata_service/restapi/tests/test_argo.py:201]: same here `geneticspecimens` but no `geneticspecimens-list`

File: chord_metadata_service/restapi/tests/test_argo.py (hunk 1 above)
Comment (daisieh) [reply chord_metadata_service/restapi/tests/test_argo.py:223]: I don't know what it's supposed to be, but it seems to work now?

**Review by amanjeev — APPROVED** [review]
hooray!

**Pull request merged**